packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -1083,7 +1083,7 @@ const New = decorate([
                     form={state.formId}
                     handler={effects.editJob}
                     icon='save'
-                    redirectOnSuccess={state.isJobInvalid ? undefined : '/backup'}
+                    redirectOnSuccess={state.isJobInvalid ? undefined : ActionButton.GO_BACK}
```

Comment:
Should we update the create button as well?

Reply:
I think we should use the same approach for creating and updating, so yes

Reply:
So it would bring you back to wherever you were before, so if you were, let's say, on a VM Console and clicked on Backup > New from the menu, then at the end of the creation, it would bring you back to the VM Console. I'm not sure that's what makes the most sense in that case. Ideally, it should call `goBack()` only when we're coming from the Overview page, but I'm not sure that's easily feasible.

Reply:
Furthermore, when creating a backup job, there is an intermediate screen to choose the type of backup, maybe it only makes sense when editing a job then.